### PR TITLE
adding r-nlsic

### PR DESCRIPTION
**Pull request by sgsokol**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sgsokol** [comment]
Hi @conda-forge/help-r

Tests of my R package nlsic are failing on all 3 platforms within simple "library('nlsic')". Error message says
```
Error in library("nlsic") : there is no package called ‘nlsic’
```
which should not be as there was no error during the building stage.
Could you have a look, please?

Thanks,
Serguei.

File: recipes/r-nlsic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-nnls
+  run:
+    - r-base
+    - r-nnls
+    - r-numderiv
```
Comment (dbast) [inline recipes/r-nlsic/meta.yaml:33]: Where does that come from?
```suggestion
```

**Comment by dbast** [comment]
You also have to commit the build.sh/bld.bat files generated by the skeleton.

File: recipes/r-nlsic/meta.yaml (hunk 1 above)
Comment (sgsokol) [reply recipes/r-nlsic/meta.yaml:33]: r-base and r-nnls were generated by skeleton (NNLS is a package imported to nlsic so its presence is indeed mandatory).
I have added r-numderiv as this package is in Suggestions of nlsic.

Comment (dbast) [reply recipes/r-nlsic/meta.yaml:33]: please remove: while it is possible to add suggested packages as run dependency, the conda-forge cran packages generally don't do that. So the user can decide, if he wants the extra package installed or not.

Comment (sgsokol) [reply recipes/r-nlsic/meta.yaml]: done

**Comment by sgsokol** [comment]
@dbast Thanks for reviewing and advising!

**Pull request merged**